content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -4,198 +4,139 @@ Start receiving tickets
+   the database, the page may need to be saved manually and refreshed before any further steps can
+   be taken.
 
-With the :guilabel:`Live Chat` setting engaged, click on :guilabel:`View Channels`. Then, on the
-:guilabel:`Website Live Chat Channels` dashboard, select the kanban card for the channel that was
-created for the *Helpdesk* team, or create a :guilabel:`New` one if necessary. When a kanban card is
-selected, additional options await on the channel's form.
+After the :guilabel:`Live Chat` setting is enabled on a *Helpdesk* team, a new *Live Chat* channel
+is created. Click on :guilabel:`Configure Live Chat Channel` to update the channel's settings.
 
 Customize the live chat channel
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When an individual channel is clicked on the :guilabel:`Website Live Chat Channels` dashboard, Odoo
-directs the page to the channel form. From there, the :guilabel:`Channel Name` can be edited,
-however Odoo names this to match the *Helpdesk* team's kanban pipeline, by default.
+On the channel's settings page, :guilabel:`Channel Name` can be edited, though, Odoo names the
+channel to match the *Helpdesk* team name, by default.
 
 .. example::
-   If a *Helpdesk* team is named `Customer Care`, a *Live Chat* channel will be created with the
-   same name.
+   If a *Helpdesk* team is named `Customer Care`, a *Live Chat* channel is created with the same
+   name.
 
-   .. image:: receiving_tickets/receiving-tickets-live-chat-join-channel.png
+   .. image:: receiving_tickets/receiving-tickets-live-chat-new-channel.png
       :align: center
-      :alt: View of the kanban cards for the available Live Chat channels.
+      :alt: View of the Kanban cards for the available Live Chat channels.
 
 On the channel form, navigate through the tabs to complete the setup.
 
 Add operators
 *************
 
-*Operators* are the users who will act as agents and respond to live chat requests from customers.
-The user who originally created the live chat channel will be added by default.
+*Operators* are the users who act as agents and respond to live chat requests from customers. The
+user who created the live chat channel is added by default.
 
-To add additional users, navigate and click on the live chat channel from the :guilabel:`Website
-Live Chat Channels` dashboard, and on the :guilabel:`Operators` tab, click :guilabel:`ADD`.
+To add additional users, click on the :guilabel:`Operators` tab, click :guilabel:`ADD`.
 
-Then, click the check box next to the users to be added, and click :guilabel:`SELECT`.
-:guilabel:`New` operators can be created and added to the list, as well, by filling out the
-:guilabel:`Create Operators` form and then clicking :guilabel:`SAVE & CLOSE` (or :guilabel:`SAVE &
-NEW` for multiplerecord creations).
+Click the checkbox next to the users to be added, and click :guilabel:`SELECT`. Click
+:guilabel:`New` to create new operators. Then, click :guilabel:`SAVE & CLOSE`, or :guilabel:`SAVE &
```

Comment:
Same with the Save & Close - they are all title case. Sorry I didn't catch this the first time!